site/content/blog/2022-03-01-whats-new-in-svelte-march-2022.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "What's new in Svelte: March 2022"
+description: "Svelte Summit Spring is coming... and page endpoints are here!"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+Just announced: [Svelte Summit Spring](https://www.sveltesummit.com/) will be taking place on April 30, 2022. The 5th Virtual Svelte Conference is [looking for speakers](https://www.sveltesummit.com/#speakers) and [sponsors](https://www.sveltesummit.com/sponsors)... so it's time to dust off those proposals!
+
+Also, some long-requested features were added to SvelteKit this month... including page endpoints! This change in how `load` functions makes it easier to fetch data required for basic pages, redirect from POST responses and handle 404s and other errors.
```

Comment:
"This change in how ..." - I have a hard time figuring out what the sentence means. I'm no native speaker, but I feel it's grammatically incorrect

Reply:
Also, some long-requested features were added to SvelteKit this month... including page endpoints!

This change in `load` functions can make it easier to:
 - fetch data required for basic pages
 - redirect from POST responses
 - handle 404s and other errors

Reply:
I think a part of the difficulty in reading it probably comes from the fact that we're used to saying `load functions` because `load` is actually a `function` exported from pages,

From what I understood, what it's trying to say is "how the load function ~functions~ works", I might've misread it

Reply:
> I think a part of the difficulty in reading it probably comes from the fact that we're used to saying `load functions` because `load` is actually a `function` exported from pages,
> 
> From what I understood, what it's trying to say is "how the load function ~functions~ works", I might've misread it

Ah yes, in which case it might say "how `load` functions work"